WhaleSpotting/react-app/src/components/Map.tsx

```diff
@@ -0,0 +1,28 @@
+import "../styles/Map.scss";
+import "../styles/Home.scss";
+import React, { useState } from "react";
+import MapChart from "./MapChart";
+import SightingMapInfo from "./SightingMapInfo";
+
+export interface Chosen {
+    id: number,
+    lat: number,
+    lon: number
+}
+
+export default function Map(): JSX.Element {
+
+    const [chosen, setChosen] = useState<Chosen>();
+
+    return (
+        <div className="map-component" data-testid="map-component">
+            <h2 className="map-header">MAP OF SIGHTINGS</h2>
```

Comment:
This would be the only part of the sight with all caps, should we have this as sentence case instead?